source/locale/en/symbols.dic

```diff
@@ -112,22 +112,18 @@ _	line	most
 ◾	black square	some
 □	white square	some
 ◦	white bullet	some
-⇒	right double arrow	some

```

Comment:
Changing this to "implies" might be controversial. It makes sense in the domain of mathematics but arrows like this can be used in other locations. The intent may not be clear anymore.

Reply:
Could you give an example? according to my research this is mostly used in formulas, mostly in mathematics but also in physics or other natural science fields, but I didn't see any other use case for this out there. In any formula this is interpreted as "implies".

Reply:
Another way to solve such problems could be a new checkbox in the interpunctuation dialog in NVDA which allows you to use synthesizer's pronounciation or not. If disabled, NVDA would pronounce the symbol as it is in the symols.dic, and if it is enabled NVDA would pronounce it as it comes from the synthesizer, ignoring the patern in the symbols.dic.

Reply:
I have to agree with @feerrenrut here. Two particular examples which comes to my mint are various posts where  this arrows shows the fact that the event to the left is connected to the event to the right and formal grammars where this arrow is used to point from shorter form of the grammar to the longer one. Intent of this symbol was certainly cleaner as an arrow.